### DISCUSSION TITLE
Lifetime token

### DISCUSSION THREAD

**Discussion by starredev** [pocketbase/pocketbase#5323] [Q&A]
I have a question, i made an ios/android app with ionic with pocketbase integrated as backend services. Is it possible to create a lifetime token so the user is still logged in (same as instagram/facebook have)? whether the user has not logged in for a week or a year?

what makes it difficult is that for the frontend the application user wants the user to log in again every day. But I could easily build in that if the json token is older than 1 day the user is logged out

**Comment by ganigeorgiev** [comment]
You can change the default tokens duration from the _Admin UI > Settings > Token options_.

There is no option (at least not out of the box) that will generate a single token with custom duration (_with the new impersonate web API in the next release it will be possible to generate a static auth token with custom duration but this api will be accessible only by superusers/admins_).
